static/queues.html

```diff
@@ -184,6 +230,67 @@ <h3>Add queue</h3>
       document.querySelector('#dataTags').onclick = e => {
         window.avalanchemq.helpers.argumentHelperJSON("arguments", e)
       }
+
+      document.querySelector("#deleteQueues").addEventListener('submit', function (evt) {
+        evt.preventDefault()
+
+        if (window.confirm('Are you sure? The selected queues are going to be deleted. Messages cannot be recovered after deletion.')) {
+          var selectedList = document.querySelectorAll('[data-select]')
```

Comment:
always use `const` or `let`, never `var`. var has weird/outdated scoping rules. in this case you want to use `const` as the variable will never change.